data/example/data/datapackage.json

```diff
@@ -0,0 +1,79 @@
+{
+    "name": "Example TIDES Data Package",
+    "title": "example",
+    "profile": "tabular-data-package",
+    "licenses": [
+        {
+            "name": "Apache-2.0"
+        }
+    ],
+    "contributors": [
+        {
+            "title": "My Name",
+            "email": "me@myself.com"
+        }
+    ],
+    "maintainers": [
+        {
+            "title": "Another Name",
+            "email": "another@myself.com"
+        }
+    ],
```

Comment:
✅  NTDID

Wondering if we should add the software to the individual [resources as `sources`](https://specs.frictionlessdata.io/data-package/#metadata)since each resource might have a different one and it might be nice to know which is which (especially when trying to grok somebody elses workflow).

Thoguhts?

Reply:
Per https://specs.frictionlessdata.io/patterns/#private-properties, this shouldn't be a private property b/c it is user specified metadata.

Reply:
For now, added:
- `ntd_id` in `datapackage.json`
- `vendor`, `product`, and `component` in `sources` in `resources`